metadata.json

```diff
@@ -10,7 +10,7 @@
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 8.1.0 < 9.0.0"
+      "version_requirement": ">= 9.0.0 < 10.0.0"
```

Comment:
concat is at v9.10. We should probably update this dependency as well in this change.

Reply:
Fixed, allowing concat 9.x